server/app.js

```diff
@@ -0,0 +1,38 @@
+require('dotenv').config()
+
+const express = require('express')
+const bodyParser = require('body-parser')
+const logger = require('./etc/logger')
+
+const app = express()
+
+app.use(express.static('public'))
+app.use(bodyParser.json())
+
+require('./routes')(app)
+
+// catch 404 and forward to error handler
+app.use((req, res) => {
+  return res.status(404).send({
+    message: 'Not Found',
+    error: {}
+  })
+})
+
+app.use((err, req, res) => {
+  const status = err.status || 500
+  if (status >= 500) {
+    logger.error(err.message)
+    logger.error(err)
```

Comment:
I think the right thing to do here when using pino is `logger.error(err, err.message)`, so that the stack and the error message are in the same line.